app/models/cleaner.rb

```diff
@@ -1,2 +1,15 @@
 class Cleaner < ActiveRecord::Base
+  EMAIL_REGEX = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/
```

Comment:
@mayur-kambariya :+1: Hope the regex is tested properly using rubular.com OR other online regex testers.